source/clear-linux/tutorials/tutorials.rst

```diff
@@ -5,18 +5,18 @@ Tutorials
 
 Our Tutorials:
 
-* Demonstrate how to use |CL| features for cloud, client, distributed
-  processing, and virtual environments
-* Involve specific application of |CL| that often use third-party tools 
-* Extend the development possibilities of |CL| with specific use cases
+* demonstrate how to use |CL-ATTR| features for cloud, client, distributed
+  processing, and virtual environments.
+* involve specific application of |CL| that often use third-party tools. 
+* extend the development possibilities of |CL| with specific use cases.
 
 This section provides detailed instructions to guide you through completing
-specific |CLOSIA| use cases.
+specific |CL| use cases.
 
 .. toctree::
    :maxdepth: 2
 
-   wordpress
+   wordpress/wordpress
```

Comment:
@krisdale. Let's discuss, in person, potential use of `:hidden:` directive. See example:
https://github.com/clearlinux/clear-linux-documentation/pull/225/files/9081f8e97d59774fa3304486c54a6a068ffb49e5